### ISSUE TITLE
Menu bar not accessible with keyboard when Integrated Terminal has focus

### ISSUE THREAD

**Issue by saelfaer** [Microsoft/vscode#12442]
- VSCode Version: 1.5.2
- OS Version: Linux

Steps to Reproduce:
1.  open integrated terminal (CTRL+`)
2. focus the terminal (click on it) 
3. press  any combination with the keyboard to try and open a menu, (ALT+V for the view menu)
4. Notice that pressing ALT does highlight the menu Underline hints, but no menu pops up when using proper keyboard navigation

**Comment by Tyriar** [comment]
Works fine for me under Ubuntu 16.04 on 1.5.2 and 1.6.0-insiders, any more details on your repro?

**Comment by saelfaer** [comment]
updated to vsc 1.5.3 yesterday, still the same problem
I can add that i'm on Linux Mint, based on ubuntu 14.04 > this is the only real difference i can spot.
other than that I can reproduce this every time i open VSC.
I'm currently looking into reproducing this on other systems, like workstations of collegues of mine. Will get back here with more info later

**Comment by Tyriar** [comment]
I still can only repro on Windows where it's as designed. Maybe you have a custom terminal keybinding that binds alt or it's an OS level thing?
